tests/helpers.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+import jubilant
+
+
+def create_workflow(
```

Comment:
[`start`](https://docs.temporal.io/cli/workflow/#start) is used to start a workflow execution, rather. I'd change the wording to reflect that (applies in various places).

Reply:
addressed in 4343511fcdbc9e94d66edc223dff2a8ec080a7d2